test/statmodels.jl

```diff
@@ -63,6 +63,73 @@ end
 @test_throws ErrorException PValue(-0.1)
 @test_throws ErrorException PValue(1.1)
 @test PValue(PValue(0.05)) === PValue(0.05)
+@test isless(PValue(0.01), 0.05)
+@test isless(PValue(0.01), NaN) == isless(0.01, NaN)
+@test (PValue(0.01) < NaN) == (0.01 < NaN)
+@test isless(NaN, PValue(0.01)) == isless(NaN, 0.01)
+@test (NaN < PValue(0.01)) == (NaN < 0.01)
+@test isequal(NaN, PValue(0.01)) == isequal(NaN, 0.01)
+@test (NaN == PValue(0.01)) == (NaN == 0.01)
+@test isequal(PValue(0.01), NaN) == isequal(0.01, NaN)
```

Comment:
A missing test is `isequal(PValue(NaN), NaN)`. Without it, `isequal` could fall back to `==` without tests failing.

Reply:
Added in #669